org.eclipse.cpacep/test/org/eclipse/cpacep/CPACEPConnectorTest.java

```diff
@@ -0,0 +1,65 @@
+package org.eclipse.cpacep;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import org.eclipse.cpacep.util.CPACEPConnector;
+import org.eclipse.cpacep.util.FileHandler;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CPACEPConnectorTest {
+	//String homePath = "/home/supun/Documents/cpachecker";
+	//String homePath = "/home/supun/Documents/CPAchecker-1.7-unix";
+	String homePath = File.separator + "home" + File.separator + "travis" + File.separator + "build" + File.separator + "SupunArunoda" + File.separator + "cpachecker";
```

Comment:
In the previous review, I told you about `Path` and `Paths`. Please try to use these to tidy up the code.